OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -120,8 +123,7 @@ function GetGSServers()
 
 	InitServerList();//needed here to prevent big accessed nones!
 	console = R6Console(Root.Console);
-	pLevel	= GetLevel();
-	class'OpenLogger'.static.Debug("CONSOLE: " $ console $ " LEVEL: " $ pLevel $ " LISTBOX: " $ m_ServerListBox, self);

```

Comment:
Removed this logger line.  GSServers() gets called on every re-sort now, and this was clogging up the logs with info we don't really need anymore.